src/decisionEngine/constants/quantitativeDecisionEngineConstant.js

```diff
@@ -0,0 +1,9 @@
+let quantitativeConstants = {
+  idealROCE: 30,
+  idealPE: 10,
+  PE_sensitivity: 0.7,
```

Comment:
Use consistent naming pattern

Reply:
You are using different naming patterns. some names with _, some names in camelCase, some names with a mixture of both. Use only only one pattern.